src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[NUMBER_OF_BALLS];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Ball();
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i]);
+        }
```

Comment:
```suggestion
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
we don't need to save balls into array, let's print a ball immediately after it has been generated